cmd/crc/cmd/daemon.go

```diff
@@ -150,7 +159,7 @@ func run(configuration *types.Configuration) error {
 		}
 	}()
 
-	networkListener, err := vn.Listen("tcp", fmt.Sprintf("%s:7777", hostVirtualIP))
+	networkListener, err := vn.Listen("tcp", fmt.Sprintf("%s:80", hostVirtualIP))
```

Comment:
If I read the PR description correctly, this will break podman 3 support? At this point I'm missing the big picture regarding when we are going to start using podman 4, so it's hard to give meaningful reviews.

Reply:
My expectation is that we'll be getting podman4 with OpenShift 4.11 and with fcos 36?

Reply:
Podman 4.x only

Reply:
@cfergeau It will not break anything in respect to openshift bundle and add support for fcos36 which going to be in the stable channel soon (as of now available in testing/next stream).

Reply:
> It will not break anything in respect to openshift bundle

Why? podman 3 from OpenShift 4.10 bundles is fine with `vn.Listen("tcp", fmt.Sprintf("%s:80", hostVirtualIP))` instead of `vn.Listen("tcp", fmt.Sprintf("%s:7777", hostVirtualIP))`?

Reply:
@cfergeau yes, because podman 3  which is part of openshift-4.x doesn't even support gvproxy bits so if you are using openshift bundle and then follow something below doesn't work and this change doesn't change it.

```
<-- start crc with openshift bundle -->
$ eval $(crc podman-env)
$ podman run -d -p 8080:80 docker.io/httpd:2.4
$ curl -IK localhost:8080
will show error.
```